librz/core/cmd_descs/cmd_descs.yaml

```diff
@@ -112,10 +112,10 @@ commands:
             arg_str: ""
             comment: List defined macros
           - text: (
-            arg_str: foo; ?e Disassemble 10 bytes at 0x10000; pd 10 @ 0x10000)
+            arg_str: foo; echo Disassemble 10 bytes at 0x10000; pd 10 @ 0x10000)
```

Comment:
Shouldn't it use quotes around `echo` argument?

Reply:
Here and in other places in this PR

Reply:
Not needed, `echo` accepts strings with spaces similar to the shell.